### PR TITLE
fixed nucleic acid analysis for Chi torsion calculation

### LINKED ISSUE #2248: Nucleic acid analysis Chi torsion calculation
Hi @orbeckst ,

I think the chi torsion calculation could use a correction. 

Currently you have this,
https://github.com/MDAnalysis/mdanalysis/blob/da5f93428f576f95379e4e971c5ca71815fbdfd5/package/MDAnalysis/analysis/nuclinfo.py#L669-L677

However, the if loop will always be skipped as all the nucleosides have those 4 atoms selected before. 
So, in case of ADENINE and GUANINE, you still get a value for O4'-C1'-N1-C2 but you want O4'-C1'-N9-C4. One might not have got much of a different value as for ADE and GUA the atoms N1-C2 and N9-C4 are lying almost in the same plane, but it's not exactly what you want.

Exchanging the selections with each other should work. 

```python
c = universe.select_atoms(" atom {0!s} {1!s} O4\' ".format(seg, i), 
                               " atom {0!s} {1!s} C1\' ".format(seg, i), 
                               " atom {0!s} {1!s} N9 ".format(seg, i), 
                               " atom {0!s} {1!s} C4  ".format(seg, i)) 
if len(c) < 4:
    c = universe.select_atoms(" atom {0!s} {1!s} O4\' ".format(seg, i), 
                               " atom {0!s} {1!s} C1\' ".format(seg, i), 
                               " atom {0!s} {1!s} N1 ".format(seg, i), 
                               " atom {0!s} {1!s} C2  ".format(seg, i)) 
```

### PR DESCRIPTION
**Pull request by aakognole**
Fixes #2248 

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [x] Tests?
 - [ ] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/CHANGELOG [hunk 1]
```diff
@@ -21,6 +21,7 @@ mm/dd/yy micaela-matta, xiki-tempula, zemanj, mattwthompson, orbeckst, aliehlen,
   * 0.20.0
 
 Enhancements
+  * fixed calculation of Chi torsion in nucleic acid analysis (PR #2248)
```
Comment (orbeckst) [inline package/CHANGELOG]: This goes under Fixes and reference the issue #2248

Comment (orbeckst) [reply package/CHANGELOG]: I fixed it and added your GitHub handle to CHANGELOG.

**Comment by orbeckst** [comment]
Thanks for your first PR. I am waiting for CI to come back with all green and will then squash-merge!

**Comment by aakognole** [comment]
Hi @orbeckst Yes, it was my first ever PR. Although it was a minor fix, glad to contribute. Thanks!

**Comment by orbeckst** [comment]
@aakognole congratulations to your first contribution! Thank you.

**Pull request merged**